src/functions/template-tags/multiday.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Calendar Multiday Functions
+ *
+ * Display functions (template-tags) for use in WordPress templates.
+ */
+
+// Don't load directly
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+if ( ! class_exists( 'Tribe__Events__Main' ) ) {
+	return;
+}
+
+
+/**
+ * Used in the month loop.
```

Comment:
* :no_entry_sign: Doc comment for parameter "$event" missing
* :no_entry_sign: Doc comment for parameter "$day_date" missing
* :no_entry_sign: Doc comment for parameter "$is_start_of_week" missing
* :no_entry_sign: Doc comment for parameter "$today_date" missing

<sub>hash: bb32be55fa55116d72ee0c13d5ae8210</sub>